README.md

```diff
@@ -220,13 +220,22 @@ contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additio
 # References
 
 V. Syrgkanis, V. Lei, M. Oprescu, M. Hei, K. Battocchi, G. Lewis.
-**Machine Learning Estimation of Heterogeneous Treatment Effects with Instruments**
-[*ArXiv preprint arXiv:1905.10176*](https://arxiv.org/abs/1905.10176)
+**Machine Learning Estimation of Heterogeneous Treatment Effects with Instruments.**
+[*ArXiv preprint arXiv:1905.10176*](https://arxiv.org/abs/1905.10176), 2019
+
+D. Foster, V. Syrgkanis.
+**Orthogonal Statistical Learning.**
+[*Proceedings of the 32nd Annual Conference on Learning Theory (COLT)*](https://arxiv.org/pdf/1901.09036.pdf), 2019
+**(Best Paper Award)**
 
 M. Oprescu, V. Syrgkanis and Z. S. Wu.
 **Orthogonal Random Forest for Causal Inference.**
-[*Proceedings of the 36th International Conference on Machine Learning*](http://proceedings.mlr.press/v97/oprescu19a.html), 2019.
+[*Proceedings of the 36th International Conference on Machine Learning, ICML'19*](http://proceedings.mlr.press/v97/oprescu19a.html), 2019.
+
+V. Chernozhukov, D. Nekipelov, V. Semenova, V. Syrgkanis.
+**Plug-in Regularized Estimation of High-Dimensional Parameters in Nonlinear Semiparametric Models.**
```

Comment:
Could you also take a look at the references in [doc/spec/references.rst](../../tree/master/doc/spec/references.rst)?  Some of the titles/links appear out of sync.

Reply:
Not sure I'll have time to check these immediately. Maybe raise an issue and commit this readme for now, and I'll get to updating the docs in the immediate future before our next release. I think  we might need a bigger doc update.

Reply:
Sure, that's fine.